src/flamenco/runtime/fd_executor.c

```diff
@@ -466,11 +466,9 @@ static void
 export_account_state( fd_borrowed_account_t * borrowed_account,
                       fd_exec_test_acct_state_t * output_account ) {
     // Address
-    output_account->has_address = true;
     fd_memcpy(output_account->address, borrowed_account->pubkey, sizeof(fd_pubkey_t));
 
     // Lamports
-    output_account->has_lamports = true;
```

Comment:
Why are we removing these?

Reply:
I removed explicitly optional types from protobuf definitions since this is default. Removing the optional keyword removes generation of `has_xxx` traits within protobuf message structs

Reply:
any implications to existing fuzzers/tests that use these?